src/gen.ts

```diff
@@ -438,7 +466,7 @@ export function gen(code: string): [(OpCode | OpValue)[], ConstantValue[]] {
     const args = expr.arguments || ([] as ts.Expression[])
     args.forEach(visitor)
     op.push(OpCode.Push)
-    op.push({ value: args.length })
+    op.push({ value: args.length, kind: 'normal' })
```

Comment:
What about an enum rather than literal

Reply:
I can do that. But I don't see the benefits. Could you describe your point of view (why enum is better in this case)?

Reply:
Never mind，not a big deal